view/frontend/layout/catalog_product_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="product.info.addtocart">
+            <block name="express.pay" template="Bold_CheckoutPaymentBooster::express-pay.phtml">
+                <arguments>
+                    <argument name="express_pay_view_model" xsi:type="object">Bold\CheckoutPaymentBooster\ViewModel\ExpressPay</argument>
+                    <argument name="render_page_source" xsi:type="const">Bold\CheckoutPaymentBooster\UI\PaymentBoosterConfigProvider::PAGE_SOURCE_PRODUCT</argument>
+                </arguments>
+            </block>
+        </referenceBlock>
+        <referenceBlock name="product.info.addtocart.additional">
+            <block name="express.pay" template="Bold_CheckoutPaymentBooster::express-pay.phtml">
+                <arguments>
+                    <argument name="express_pay_view_model" xsi:type="object">Bold\CheckoutPaymentBooster\ViewModel\ExpressPay</argument>
+                    <argument name="render_page_source" xsi:type="const">Bold\CheckoutPaymentBooster\UI\PaymentBoosterConfigProvider::PAGE_SOURCE_PRODUCT</argument>
+                </arguments>
+            </block>
+        </referenceBlock>        
+        <!-- <referenceBlock name="product.info.addtocart.additional">
+            <block name="express.pay" template="Bold_CheckoutPaymentBooster::express-pay.phtml">
+                <arguments>
+                    <argument name="express_pay_view_model" xsi:type="object">Bold\CheckoutPaymentBooster\ViewModel\ExpressPay</argument>
+                    <argument name="render_page_source" xsi:type="const">Bold\CheckoutPaymentBooster\UI\PaymentBoosterConfigProvider::PAGE_SOURCE_PRODUCT</argument>
+                </arguments>
+            </block>
+        </referenceBlock> -->
```

Comment:
It's typically not good practice to leave commented code in PRs, Git's history can normally handle this if there's a reason for keeping this